sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -172,6 +173,150 @@ describe("AssessmentStudentsController(e2e)-getAssessmentNOA", () => {
       .expect(expectation);
   });
 
+  it("Should get the student NOA details for an eligible full-time application when the student tries to access it.", async () => {
```

Comment:
Can we have the description from API perspective. ?

for e.g. `Should get the student NOA details for a full-time application when the application assessment is created.`